### ISSUE TITLE
'Immediate window' scoped to method Alive is running.

### ISSUE THREAD

**Issue by Ijwu** [CodeConnect/AliveFeedback#78]
It would be fancy to have a REPL/Immediate window with the scope of the method being run by Alive. I realize Alive runs the code asynchronously and displays the values so it's not exactly always keeping a live scope around. However, I think it would be helpful to be able to run a method to completion and allow Alive to retain the scope, if it doesn't already, and allow for some off-the-cuff scripting.

**Comment by AmadeusW** [comment]
I'm not sure if I can imagine your use case. What would you like to achieve?
Alive is _like REPL_ in a way that you can just write code at the bottom of your method for off-the-cuff scripting.

**Comment by Ijwu** [comment]
It is to my understanding that Alive re-evaluates, re-compiles, and re-runs the method upon each change. If the method involves loading data from a file or any potentially slow or bottle-necking operation then simply adding code at the bottom of the method forces a re-run and you end up with slow evaluation. Let me know if I'm fundamentally misunderstanding how Alive works. If I'm wrong then I guess this isn't really an issue.

If I'm right, however, it would be useful to be able to maintain a live scope of a single run-through of a method. That way I can load my data from a file and do some scripting on it for quicker debugging.

**Comment by AmadeusW** [comment]
You're absolutely right about how Alive works. That's great to see that you thought the same way we do :) This is our long term view for Alive. Currently we have more pressing issues to resolve, but we always keep in mind the long term goal.

**Comment by Ijwu** [comment]
It's definitely not a pressing or important issue. Just a feature request, if anything. I'm glad to know it's in mind at least!
